spec/features/user_views_lists_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'User views all lists' do
+  scenario 'successfully' do
+    create(:list, title: 'Best Sci Fi')
+    book = create(:book, cover_image_url: 'http://placehold.it/140x100')
```

Comment:
@evan-007 @samnang  i think the gist got cut off.  

I setup the lists.rb factory like this.  Thoughts?

```ruby
FactoryGirl.define do
  factory :list do
    title 'MyString'
    description 'MyText'

    factory :list_with_books do

      transient do
        books_count 2
      end

      after(:create) do |list, evaluator|
        create_list(:book, evaluator.books_count, list: list)
      end
    end

  end
end
```